source/modules/collection-api/event/actions/create.md.erb

```diff
@@ -1,27 +1,35 @@
 ### Event Creation
 
-```php
-<?php
-
-use Inspetor;
-
-$inspetor = $this->getConfiguredInspetor();
-
-$inspetor->trackEventCreation($event);
-
-?>
+```sh
+curl -X POST https://collection-prod.inspcdn.net/event \
+    -H "Authorization: Bearer $API_KEY" \    
+    -d '{
+        "id":          "12345",
+        "name":        "Reveillon 2020",
+        "is_physical": true,
+        "dates":  [
+            {
+                "id":       "1",
+                "datetime": 1574210000
+            },
+            {
+                "id":       "2",
+                "datetime": 1574315000
+            }
+        ],
+        "creator_id":  "22342",
+        "admin_ids":   ["22342", "23536"]
+        "address":     {
+            "street_name":   "Rua Fidalga",
+            "street_number": "252",
+            "zip_code":      "05432-010",
+            "city":          "São Paulo",
+            "state":         "SP",
+            "country":       "BR"
+        },
+        "description": "Lorem Ipsum",
```

Comment:
Nit: Since we are adding all fields here I think it would be nice to add the `status`